src/libcollections/str.rs

```diff
@@ -599,7 +599,7 @@ impl str {
     /// # #![feature(str_char, core)]
     /// use std::str::CharRange;
     ///
-    /// let s = "中华Việt Nam";
+    /// let s = "中华Việt Nam";
```

Comment:
The diff is not broken, the source was changed to use "decomposed" code points (an ASCII `e` followed by combining code points, rather than `ệ` as a single code point) as the "This outputs" change below indicates.